packages/app-graphql-playground/README.md

```diff
@@ -0,0 +1,44 @@
+<!--[meta]
+section: api
+subSection: apps
+title: GraphQL Playground App
+draft: true
+[meta]-->
+
+# GraphQL Playground App
+
+A Keystone App that creates an Apollo GraphQL playground.
+
+## Usage
+
+FIXME
```

Comment:
I don't want todo's or fixme in docs. I just spent ages removing all of them. 

When published it makes the website look broken. It's better to just not include these sections. Or to take a moment to write them.

Reply:
Ooh, good catch, I had fully intended to do these docs on this PR and completely forgot :-( Will fix.